### PR TITLE
Add tubeszb router support

### PR DESCRIPTION
**Pull request by tube0013**
Adds tubeszb router support. new fw build for tubeszb routers, converter based on ti.router

### CODE REVIEW

File: devices/tubeszb.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const reporting = require('../lib/reporting');
+
+module.exports = [
+    {
+        zigbeeModel: ['tubeszb.router'],
+        model: 'tubeszb.router',
+        vendor: 'TubesZB',
+        description: '[TubesZB CC2652 Router](https://github.com/tube0013/tube_gateways/tree/main/tube_cc_router)',
```
Comment (Koenkk) [inline devices/tubeszb.js:9]: On https://github.com/tube0013/tube_gateways/blob/main/tube_cc_router/README.md you mention:

> FW is from the Z-Stack Router FW Repository: https://github.com/Koenkk/Z-Stack-firmware/tree/master/router/Z-Stack_3.x.0/bin

why does this need separate support?

Comment (tube0013) [reply devices/tubeszb.js:9]: I've started compiling my own FW with some small changes including model id.

Comment (Koenkk) [reply devices/tubeszb.js:9]: where can this fw be found? Don't see it on your github, it would be good to also open source the changes you made.

Comment (tube0013) [reply devices/tubeszb.js:9]: Absolutely planning on putting at at the link above. I am just short on time and a few routers got out with this new FW before I remembered I needed a converter (I use zha). So I was scrambling a bit.

Comment (Koenkk) [reply devices/tubeszb.js:9]: Good, will merge this once the fw files + changes are available on the linked github

Comment (tube0013) [reply devices/tubeszb.js:9]: I just posted the FW and updated the Readme to be more specific about this change. Will document the full changes as soon as I get a chance.

Comment (Koenkk) [reply devices/tubeszb.js:9]: thanks

Comment (tube0013) [reply devices/tubeszb.js]: Changes are documented: https://github.com/tube0013/tube_gateways/tree/main/tube_cc_router

**Pull request merged**